src/_Problems_/binary-tree-to-binary-search-tree/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Given a Binary Tree, convert it to a Binary Search Tree. 
+ * The conversion must be done in such a way that keeps the original structure of Binary Tree.
+ * Example 1
+Input:
+          10
+         /  \
+        2    7
+       / \
+      8   4
+Output:
+          8
+         /  \
+        4    10
+       / \
+      2   7
+ */
+
+// Helper function to store inorder traversal of a binary tree
+function storeInorder(root, inorder) {
+  // Base case
+  if (!root) return;
+
+  // First store the left subtree
+  storeInorder(root.left, inorder);
+
+  // Append root's data
+  inorder.push(root.data);
+
+  // Store right subtree
+  storeInorder(root.right, inorder);
+}
```

Comment:
Try to return an array instead of accepting an array as an argument. For help, you can go through the various traversal algorithms and problems on BST from the README